products/motorola.md

```diff
@@ -0,0 +1,366 @@
+    eol: 2023-11-01
+    link: https://www.motorola.com/we/smartphones-moto-g-51-5g/p
+-   releaseCycle: "moto-g50-5g"
+    releaseLabel: "moto g50 5g"
+    releaseDate: 2021-08-01
+    eol: 2023-08-01
+    link: https://nz.motorola.com/smartphones-motorola-g50-5g/p
+-   releaseCycle: "moto-g50"
+    releaseLabel: "moto g50"
+    releaseDate: 2021-03-01
+    eol: 2023-03-01
+    link: https://www.motorola.com/we/smartphones-moto-g-50/p
+-   releaseCycle: "moto-g42"
+    releaseLabel: "moto g42"
+    support: true
+    releaseDate: 2022-06-01
+    eol: 2025-06-01
+    link: https://www.motorola.com/we/smartphones-moto-g-42/p
+-   releaseCycle: "moto-g41"
+    releaseLabel: "moto g41"
+    releaseDate: 2021-11-01
+    eol: 2023-11-01
+    link: https://www.motorola.com/we/smartphones-moto-g-41/p
+-   releaseCycle: "moto-g40-fusion"
+    releaseLabel: "moto g40 FUSION"
+    releaseDate: 2021-05-01
+    eol: 2023-05-01
+    link: https://www.motorola.in/smartphones-moto-g40-fusion/p
+-   releaseCycle: "moto-g32"
+    releaseDate: 2022-08-01
+    eol: 2025-08-01
+    support: true
+    link: https://www.motorola.in/smartphones-moto-g32/p
+-   releaseCycle: "moto-g31"
+    releaseLabel: "moto g31"
+    releaseDate: 2021-11-01
+    eol: 2023-11-01
+    link: https://www.motorola.com/we/smartphones-moto-g-31/p
+-   releaseCycle: "moto-g30"
+    releaseLabel: "moto g30"
+    releaseDate: 2021-02-01
+    eol: 2023-02-01
+    link: https://www.motorola.com/we/smartphones-moto-g-30/p
+-   releaseCycle: "moto-g22"
+    releaseLabel: "moto g22"
+    releaseDate: 2022-03-01
+    eol: 2025-03-01
+    link: https://www.motorola.com/we/smartphones-moto-g-22/p
+-   releaseCycle: "moto-g20"
+    releaseLabel: "moto g20"
+    releaseDate: 2021-05-01
+    eol: 2023-05-01
+    link: https://www.motorola.com/we/smartphones-moto-g-20/p
+-   releaseCycle: "moto-g10-power"
+    releaseLabel: "moto g10 power"
+    releaseDate: 2021-03-01
+    eol: 2023-03-01
+    link: https://www.motorola.in/smartphones-moto-g-10-power/p
+-   releaseCycle: "moto-g10"
+    releaseLabel: "moto g10"
+    releaseDate: 2021-03-01
+    eol: 2023-03-01
+    link: https://www.motorola.com/we/smartphones-moto-g-10/p
+-   releaseCycle: "moto-g-stylus-2021"
+    releaseLabel: "moto g stylus (2021)"
+    releaseDate: 2021-01-01
+    eol: 2023-01-01
+    link: https://www.motorola.com/us/smartphones-moto-g-stylus-gen-2/p
+-   releaseCycle: "moto-g-pure-2021"
+    releaseLabel: "moto g pure (2021)"
+    releaseDate: 2021-11-01
+    eol: 2023-11-01
+    link: https://www.motorola.com/us/smartphones-moto-g-pure/p
+-   releaseCycle: "moto-g-power-2021"
+    releaseLabel: "moto g power (2021)"
+    releaseDate: 2021-01-01
+    eol: 2023-01-01
+    link: https://www.motorola.com/us/smartphones-moto-g-power/p
+-   releaseCycle: "moto-g-play-2021"
+    releaseLabel: "moto g play (2021)"
+    releaseDate: 2021-01-01
+    eol: 2023-01-01
+    link: https://www.motorola.com/we/smartphones-moto-g-play/p
```

Comment:
404, could be replaced by https://www.motorola.com/us/smartphones-moto-g-play/p ?